src/k8s-configuration/azext_k8s_configuration/_help.py

```diff
@@ -6,17 +6,16 @@
 
 from knack.help_files import helps  # pylint: disable=unused-import
 
-
 helps['k8s-configuration'] = """
     type: group
-    short-summary: Commands to manage Kubernetes configuration.
+    short-summary: Commands to manage resources from Microsoft.KubernetesConfiguration.
 """
 
 helps['k8s-configuration create'] = """
     type: command
-    short-summary: Create a Kubernetes configuration.
+    short-summary: Create a Flux v1 Kubernetes configuration (This command is for Flux v1, to use the newer Flux v2, run "az k8s-configuration flux create").
```

Comment:
nitpick: change create to creates. Would be good to have the verbs in plural tense, same with the other help messages/summaries.